test/testinput/bufrtest_python_test.py

```diff
@@ -237,6 +238,15 @@ def test_highlevel_cache():
     if bufr.DataCache.has(DATA_PATH, YAML_PATH):
         assert False, "Data Cache still contains entry."
 
+def test_zarr_encoder():
+    DATA_PATH = 'testdata/gdas.t18z.1bmhs.tm00.bufr_d'
+    YAML_PATH = 'testinput/bufrtest_mhs_basic_mapping.yaml'
+    OUTPUT_PATH = 'testrun/bufrtest_python_test.zarr'
+
+    container = bufr.Parser(DATA_PATH, YAML_PATH).parse()
+
+    dataset = next(iter(zarr.Encoder(YAML_PATH).encode(container, OUTPUT_PATH).values()))
+    assert abs(dataset['ObsValue/brightnessTemperature'][0,0] - 215.89) < 1e-3
```

Comment:
Are hardwired constant values, e.g. 215.89, acceptable in the code? Just curious from coding standards perspective.

Reply:
This is just CTest code, so constants are ok as I am testing a specific use case with a specific file.